frontend/catalyst/compiler.py

```diff
@@ -294,7 +282,7 @@ class Enzyme(PassPipeline):
     """Pass pipeline to lower LLVM IR to Enzyme LLVM IR."""
 
     _executable = get_executable_path("llvm", "opt")
-    enzyme_path = get_enzyme_path("enzyme", "ENZYME_DIR")
+    enzyme_path = get_lib_path("enzyme", "ENZYME_DIR")
```

Comment:
What is the relationship between `ENZYME_DIR` and `ENZYME_BUILD_DIR`? Are they different variables?

Reply:
I think this follows a similar scheme as with our other dependencies. `ENZYME_BUILD_DIR` is the build directory when building from source, only used in our make files and cmake configs. `ENZYME_DIR` is a user variable to be able to specify the location of the compiled lib at runtime. Looking at the other dependencies, they are typically named `*_LIB_DIR` (see `MLIR_LIB_DIR` or `RUNTIME_LIB_DIR`), so maybe we could rename `ENZYME_DIR` to `ENZYME_LIB_DIR`.

Reply:
```suggestion
    enzyme_path = get_lib_path("enzyme", "ENZYME_LIB_DIR")
```

Reply:
In this case I should check where in the docs we talk about this variable.

Reply:
Changed in [d4c2b7f](https://github.com/PennyLaneAI/catalyst/pull/210/commits/d4c2b7f8b0e7005de24b505dfe71104b4a1796ba)